Subject: include cstring where necessary

Fix missing declarations while building with gcc-4.3.2 by including the
proper header file.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 demos/gtk-demo/demowindow.cc                   |    1 +
 demos/gtk-demo/textwidget.cc                   |    2 ++
 examples/book/custom/custom_widget/mywidget.cc |    1 +
 gtk/gtkmm/scale.cc                             |    2 ++
 gtk/gtkmm/targetentry.cc                       |    2 +-
 5 files changed, 7 insertions(+), 1 deletion(-)

Index: gtkmm-2.12.1/gtk/gtkmm/scale.cc
===================================================================
--- gtkmm-2.12.1.orig/gtk/gtkmm/scale.cc
+++ gtkmm-2.12.1/gtk/gtkmm/scale.cc
@@ -33,6 +33,8 @@
 #include <gtk/gtkvscale.h>
 #include <math.h>
 
+#include <cstring>
+
 namespace Gtk
 {
 
Index: gtkmm-2.12.1/examples/book/custom/custom_widget/mywidget.cc
===================================================================
--- gtkmm-2.12.1.orig/examples/book/custom/custom_widget/mywidget.cc
+++ gtkmm-2.12.1/examples/book/custom/custom_widget/mywidget.cc
@@ -20,6 +20,7 @@
 #include <gdkmm/drawable.h>
 #include <gdkmm/general.h>  // for cairo helper functions
 #include <iostream>
+#include <cstring>
 //#include <gtk/gtkwidget.h> //For GTK_IS_WIDGET()
 
 
Index: gtkmm-2.12.1/gtk/gtkmm/targetentry.cc
===================================================================
--- gtkmm-2.12.1.orig/gtk/gtkmm/targetentry.cc
+++ gtkmm-2.12.1/gtk/gtkmm/targetentry.cc
@@ -21,7 +21,7 @@
  */
 
 #include <gtkmm/targetentry.h>
-
+#include <cstring>
 
 namespace Gtk
 {
Index: gtkmm-2.12.1/demos/gtk-demo/textwidget.cc
===================================================================
--- gtkmm-2.12.1.orig/demos/gtk-demo/textwidget.cc
+++ gtkmm-2.12.1/demos/gtk-demo/textwidget.cc
@@ -19,6 +19,8 @@
  * Software Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
  */
 
+#include <cstring>
+
 #include "textwidget.h"
 #include "pangomm/fontdescription.h"
 
Index: gtkmm-2.12.1/demos/gtk-demo/demowindow.cc
===================================================================
--- gtkmm-2.12.1.orig/demos/gtk-demo/demowindow.cc
+++ gtkmm-2.12.1/demos/gtk-demo/demowindow.cc
@@ -30,6 +30,7 @@
 #include <vector>
 #include <cctype>
 #include <cerrno>
+#include <cstring>
 #include <stdio.h>
 
 #ifdef NEED_FLOCKFILE_PROTO
